libs/python.json

```diff
@@ -157,7 +160,8 @@
   {"id":"py.pymongo","imports":["pymongo"],"name":"pymongo","repo":"mongodb/mongo-python-driver","tags":["mongodb"],"tech":["storage"],"status":"awaiting-model"},
   {"id":"py.pymysql","imports":["pymysql"],"name":"PyMySQL","repo":"PyMySQL/PyMySQL","tags":["mysql"],"tech":["storage"],"status":"awaiting-model"},
   {"id":"py.redis","imports":["redis"],"name":"redis","repo":"andymccurdy/redis-py","tags":["nosql"],"tech":["storage"],"status":"model-ready"},
-  {"id":"py.sqlalchemy","imports":["sqlalchemy"],"name":"sqlalchemy","repo":"zzzeek/sqlalchemy","tags":["sql"],"tech":["storage"],"status":"model-ready"},
+  {"id":"py.sqlalchemy","imports":["sqlalchemy"],"name":"SQLAlchemy","repo":"sqlalchemy/sqlalchemy","tags":["sql"],"tech":["storage"],"status":"awaiting-model"},
+  {"id":"py.sqlalchemy-zzzeek","imports":["sqlalchemy"],"name":"SQLAlchemy","repo":"zzzeek/sqlalchemy","tags":["sql"],"tech":["storage"],"status":"model-ready"},
```

Comment:
we don't need both sqlalchemy/sqlalchemy and zzzeek/sqlalchemy. could you remove zzzeek?